Tests/Mapping/Driver/AttributeDriverTest.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the BazingaGeocoderBundle package.
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @license    MIT License
+ */
+
+namespace Bazinga\GeocoderBundle\Tests\Mapping\Driver;
+
+use Bazinga\GeocoderBundle\Mapping\Annotations\Address;
+use Bazinga\GeocoderBundle\Mapping\Annotations\Geocodeable;
+use Bazinga\GeocoderBundle\Mapping\Annotations\Latitude;
+use Bazinga\GeocoderBundle\Mapping\Annotations\Longitude;
+use Bazinga\GeocoderBundle\Mapping\Driver\AttributeDriver;
+use Bazinga\GeocoderBundle\Mapping\Exception\MappingException;
+use Doctrine\Common\Annotations\Reader;
+use PHPUnit\Framework\TestCase;
+use Symfony\Bridge\PhpUnit\SetUpTearDownTrait;
+
+/**
+ * @author Pierre du Plessis <pdples@gmail.com>
+ */
+class AttributeDriverTest extends TestCase
```

Comment:
```suggestion
final class AttributeDriverTest extends TestCase
```

Reply:
Not sure what the benefit is here for making a test class `final`, IMHO I don't think it provides anything useful. None of the other test classes is final as well Nevertheless, I have made the change

Reply:
I have a commit that I didn't made a PR yet where all tests are made final. Thanks